src/puppetlabs/puppetdb/mq.clj

```diff
@@ -45,6 +45,17 @@
   [broker megabytes]
   (set-usage!* broker megabytes #(.getStoreUsage %) "StoreUsage"))
 
+(defn- set-memory-usage!
+  "Configures the `MemoryUsage` setting for an instance of `BrokerService`.
+
+  `broker`     - the `BrokerService` to configure
+  `megabytes ` - the maximum amount of memory usage to allow for the BrokeService,
+  or `nil` to use the default value of 1GB.
+
+  Returns the (potentially modified) `broker` object."
+  [broker megabytes]
```

Comment:
Happy to leave this as is, though I've been tending away from javadoc-style docstrings in most cases, tending toward Clojure's style, i.e. tending toward strings like "Sets the maximum number of megabytes that the broker is allowed to use.  If megabytes is nil, then the broker will revert to its default limit (currently 1GB).  Returns the updated broker."